src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsAllocateStep.java

```diff
@@ -48,6 +51,26 @@ public void setUpstream(String upstream) {
         this.upstream = fixEmptyAndTrim(upstream);
     }
 
+    @CheckForNull
+    public BuildSelector getSelector() {
+        return selector;
+    }
+
+    @DataBoundSetter
+    public void setSelector(BuildSelector selector) {
+        this.selector = selector;
+    }
+
+    @CheckForNull
+    public String getParameters() {
+        return parameters;
+    }
+
+    @DataBoundSetter
+    public void setParameters(String parameters) {
```

Comment:
Would it be possible to put these parematerFilters into the Selector directly? The current API location is not very good  IMHO